lib/tasks/package.rake

```diff
@@ -0,0 +1,107 @@
+require 'tempfile'
+require 'English'
+
+def check_git
+  modified = `git ls-files -m --exclude-standard .`
+  if !modified.empty?
```

Comment:
why not `modified.any?`

Reply:
`modified` is a `String`, which does not have `any?`. I will change it to positive evaluation though.